plugins/lookup/tss.py

```diff
@@ -211,13 +241,28 @@ def from_params(**server_parameters):
         else:
             return TSSClientV0(**server_parameters)
 
-    def get_secret(self, term):
+    def get_secret(self, term, fetch_file_attachments, file_download_path):
         display.debug("tss_lookup term: %s" % term)
-
         secret_id = self._term_to_secret_id(term)
         display.vvv(u"Secret Server lookup of Secret with ID %d" % secret_id)
 
-        return self._client.get_secret_json(secret_id)
+        if fetch_file_attachments:
+            obj = self._client.get_secret(secret_id, fetch_file_attachments)
+            for i in obj['items']:
+                if os.path.isdir(file_download_path):
```

Comment:
You need some sane fallback in case `file_download_path` is `None` or `''` - in both cases `os.path.isdir()` will fail. (With `None`, `os.path.join` will probably also cause problems.)

Reply:
Handled  None or '' value